components/Tour/stagesMapView.tsx

```diff
@@ -0,0 +1,96 @@
+import React, { useState, useEffect, ReactElement } from "react";
+import { Stage, Location, Tour } from "@/model/model";
+import { withObservables } from "@nozbe/watermelondb/react";
+import { getAllStagesByTourIdQuery } from "@/services/data/stageService";
+import { getAllLocationsByStageId } from "@/services/data/locationService";
+import MapboxGL from "@rnmapbox/maps";
+import { Text } from "@ui-kitten/components";
+import StageMapLine from "@/components/Tour/StageMapLine";
+import { useNavigation } from "expo-router";
+
+// Define the StagesMapView component
+const StagesMapView = ({
+  stages,
+  tour,
+}: {
+  stages: Stage[];
+  tour: ReactElement;
+}) => {
```

Comment:
```suggestion
const StagesMapView = ({
  stages,
}: {
  stages: Stage[];
}) => {
```
tour kann nicht übergeben werden